composer/algorithms/mixup/mixup.py

```diff
@@ -57,6 +58,9 @@ def mixup_batch(x: Tensor,
         y: target tensor of shape (B, f1, f2, ..., fm), B is batch size, f1-fn
             are possible target dimensions.
         interpolation_lambda: amount of interpolation based on alpha.
```

Comment:
Can you elaborate on the docstring that it can be set to None / what that means if it's set to None?